### ISSUE TITLE
Move bot controls from multi pane to navbar

### ISSUE THREAD

**Issue by Mohammad699** [freqtrade/frequi#1964]
It will be accessible from any where in the UI because sometimes you need to restart the bot or stop it or whatever you want, you don't need to go to trade page anymore.

Dashboard: 
![image](https://github.com/freqtrade/frequi/assets/12565380/af60eded-53e9-422b-879e-e66db4250ee7)

Trade:

![image](https://github.com/freqtrade/frequi/assets/12565380/4ad7beba-897a-4993-81a3-98c6979c3f60)

Chart:

![image](https://github.com/freqtrade/frequi/assets/12565380/0db4a782-229e-4b67-ae88-1713f722b73d)

Logs:
![image](https://github.com/freqtrade/frequi/assets/12565380/f0b910c7-aea4-43b5-bccb-5a66e2979c32)

Comment the element in TradingView.vue:

![image](https://github.com/freqtrade/frequi/assets/12565380/fef0186e-3e58-4de9-88c2-fbe4e08f2214)

Add it in NavBar.vue with a condition of `botStore.hasBots` so it'll be visible only after login:

![image](https://github.com/freqtrade/frequi/assets/12565380/8827c07a-1d2f-4110-a1d5-ed75058040fd)

I don't know how to make PR on Github so I can't contribute it with PRs

And latest dev code has an issue with local host I'll post it in another issue
Thank you very much

**Comment by xmatthias** [comment]
A guide for the github PR workflow can be found [here](https://guides.github.com/introduction/flow/).

---

Now i'm however not sure this is a good approach - as it'll make the context (what's going to happen if i press this) no longer clear.
It'll work in single-bot mode (how you seem to be using it, based on the screenshots) - but freqUI supports multiple bots - the controls will however only work for one bot.

If you're looking at the dashboard and think "I need to stop bot2" - you'll very likely risk stopping the wrong bot - as the wrong bot is active.
The trade pane on the other hand is in "single bot" mode.

The only way i can think of for this is to disable it in dashboard mode if more than one bot is available - which i think is rather inconsistent from a UX perspective.


Unless you've also got an idea how to fix this in a very clear way (especially for buttons like force-enter and force-exit all) - i'll have to treat this as a question and something we'll not be adding / accepting.
